openwisp_controller/config/migrations/0011_add_relevant_permissions_to_admin_and_operators.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations 
+from django.models import Permission 
+
+
+def assignPerm(apps, schema_editor): 
+    Group= apps.get_model('openwisp_user', 'Group')
+    admin = Group.objects.get(name="Administrator")
+    operator= Group.objects.get(name="Operator")
+    operators_and_admins_can_change=["device", "config", "template", "location", "floorplan",]
+    operators_read_only_admins_manage=["vpn", "ca", "certificate",]
+    manage_operations=["add", "change", "delete"]

```

Comment:
There are inconsistencies with the spacing in between ```=```. Also, I think the variables can be cleaner, just use ```can_manage``` and ```read_only```. With a comment saying ```# Administrators can manage all objects```.

Reply:
I recommend surrounding operators by spaces, as [PEP8 suggests.](https://www.python.org/dev/peps/pep-0008/#other-recommendations)